ee/clickhouse/queries/session_recordings/test/test_clickhouse_session_recording_list.py

```diff
@@ -20,4 +24,61 @@ def _create_session_recording_event(**kwargs):
 
 
 class TestClickhouseSessionRecordingsList(ClickhouseTestMixin, factory_session_recordings_list_test(ClickhouseSessionRecordingList, _create_event, _create_session_recording_event, Action.objects.create, ActionStep.objects.create)):  # type: ignore
-    pass
+    def test_event_filter_with_person_properties(self):
+        Person.objects.create(team=self.team, distinct_ids=["user"], properties={"email": "bla"})
+        Person.objects.create(team=self.team, distinct_ids=["user2"], properties={"email": "bla2"})
+        self.create_snapshot("user", "1", self.base_time)
+        self.create_event("user", self.base_time)
+        self.create_snapshot("user", "1", self.base_time + relativedelta(seconds=30))
+        self.create_snapshot("user2", "2", self.base_time)
+        self.create_event("user2", self.base_time)
+        self.create_snapshot("user2", "2", self.base_time + relativedelta(seconds=30))
+        filter = SessionRecordingsFilter(
+            data={
+                "events": [
+                    {
+                        "id": "$pageview",
+                        "type": "events",
+                        "order": 0,
+                        "name": "$pageview",
+                        "properties": [{"key": "email", "value": ["bla"], "operator": "exact", "type": "person"}],
+                    },
+                ]
+            }
+        )
+        session_recording_list_instance = ClickhouseSessionRecordingList(filter=filter, team=self.team)
+        (session_recordings, _) = session_recording_list_instance.run()
+        self.assertEqual(len(session_recordings), 1)
+        self.assertEqual(session_recordings[0]["session_id"], "1")
+
+    def test_event_filter_with_cohort_properties(self):
+        Person.objects.create(team=self.team, distinct_ids=["user"], properties={"email": "bla"})
+        Person.objects.create(
+            team=self.team, distinct_ids=["user2"], properties={"email": "bla2", "$some_prop": "some_val"}
+        )
+        cohort = Cohort.objects.create(
+            team=self.team, name="cohort1", groups=[{"properties": {"$some_prop": "some_val"}}]
+        )
+        self.create_snapshot("user", "1", self.base_time)
+        self.create_event("user", self.base_time)
+        self.create_snapshot("user", "1", self.base_time + relativedelta(seconds=30))
+        self.create_snapshot("user2", "2", self.base_time)
+        self.create_event("user2", self.base_time)
+        self.create_snapshot("user2", "2", self.base_time + relativedelta(seconds=30))
+        filter = SessionRecordingsFilter(
```

Comment:
nit: pass team=team to any filter object you construct.

Reply:
I'm a bit out of my depth here, but when I added the team arg, I could only get the cohort filter to work if I included the `"is_simplified": True,` flag in the data object. Otherwise, it looks like this line alters the cohort filter: https://github.com/PostHog/posthog/blob/master/posthog/models/filters/filter.py#L114

My head isn't wrapped around `SimplifyFilterMixin`, any thoughts on this?

Reply:
At this point, basically all queries run the simplification logic before the query. It's a way to simplify query-building for us.

For cohorts, it might be that you're not handling all properties well. What's the specific error you're running into? This is likely indicating towards more work to be done to align this with EventQuery class.

Reply:
Ah got it - it wasn't erroring out, but it wasn't passing the tests. I think I figured it out now - looks like the simplified filter was expecting specific table + column names that were different from the joined table. Working on fixing this up now.